build.psm1

```diff
@@ -1047,11 +1056,21 @@ function Start-PSBootstrap {
             # Packaging tools
             if ($Package) { $Deps += "ruby-devel", "rpm-build", "groff" }
 
-            $PackageManager = Get-RedHatPackageManager
+            $PackageManager = Get-RedHatPackageManager -install
+
+            $currentUser = (Get-Process -Id $pid -IncludeUserName).UserName
```

Comment:
maybe use `id -u` and compare the return user id with 0 to determine if it's running as root.